dhis-2/dhis-web-api/src/main/java/org/hisp/dhis/webapi/controller/tracker/export/TrackerTrackedEntitiesExportController.java

```diff
@@ -101,17 +101,18 @@
     @Nonnull
     private final CsvEventService<TrackedEntity> csvEventService;
 
+    private final TrackedEntityFieldsParamMapper trackedEntityFieldsParamMapper;
```

Comment:
This class feels too specific. Couldn't we have a generic `fields` parameter mapper?

Reply:
We will refactor these tracker classes in another ticket. I'll focus on getting tracker to use `FieldFilteringService.filterIncludes`. So we don't (wrongly) assume anything about how field filtering might work 😅